gettsim/policy_for_date.py

```diff
@@ -60,6 +60,10 @@ def get_policies_for_date(year, tax_data_raw=None, month=1, day=1):
         tax_data["vorsorge"] = vorsorge2010
     else:
         tax_data["vorsorge"] = vorsorge_dummy
+    if year >= 2009:
+        tax_data["zve_list"] = ["nokfb", "kfb", "abg_nokfb", "abg_kfb"]
+    else:
```

Comment:
If we want to postpone #81, take out the `abg` items again here.

Reply:
Removed year dependency.